MaxText/maxengine.py

```diff
@@ -54,7 +54,7 @@
 
 class MaxEngineConfig:
   """Engine specific config class to allow using multiple MaxEngine instances in an inference run.
-  The default pyconfig.config is a global param shared across multiple instances and doesn't
+  The default pyconfig.initialize is a global param shared across multiple instances and doesn't
```

Comment:
Is this statement still correct ? initialize is a function returning a new config, right ?

Reply:
You are right! I have added a TODO to reevaluate the need for this MaxEngineConfig in the light of the restructured pyconfig.py

Reply:
Right right, but I think it might be a good idea to investigate if we can get rid of `MaxEngineConfig` altogether